src/client/features/enrichment/index.js

```diff
@@ -142,13 +153,21 @@ class Enrichment extends React.Component {
       ])
     ]);
 
+    let emptyNetwork = h('div.empty-enrichment-network', [
```

Comment:
use the empty network component instead 

https://github.com/PathwayCommons/app-ui/blob/development/src/client/common/components/empty-network.js